projects/01-cipher/README.md

```diff
@@ -222,29 +222,32 @@ usaremos las recomendaciones _por defecto_ de tanto `eslint` como `htmlhint`.
 ### Primeros pasos
 
 1. Antes que nada, asegúrate de tener un :pencil: editor de texto en
-   condiciones, algo como [Atom](https://atom.io/) o
-   [Code](https://code.visualstudio.com/).
+  condiciones, algo como [Atom](https://atom.io/) o
+  [Code](https://code.visualstudio.com/).
 2. Para ejecutar los comandos a continuación necesitarás una :shell:
-   [UNIX Shell](https://github.com/Laboratoria/bootcamp/tree/master/topics/shell),
-   que es un programita que interpreta líneas de comando (command-line
-   interpreter) así como tener [git](https://github.com/Laboratoria/bootcamp/tree/master/topics/scm/01-git)
-   instalado. Si usas un sistema operativo "UNIX-like", como GNU/Linux o MacOS,
-   ya tienes una _shell_ (terminal) instalada por defecto (y probablemente `git`
-   también). Si usas Windows puedes usar [Git bash](https://git-scm.com/download/win),
-   aunque recomendaría que consideres probar :penguin: GNU/Linux.
-3. Haz tu propio :fork_and_knife: [fork](https://help.github.com/articles/fork-a-repo/)
-   del repo de tu cohort, tus _coaches_ te compartirán un _link_ a un repo y te
-   darán acceso de lectura en ese repo.
+  [UNIX Shell](https://github.com/Laboratoria/bootcamp/tree/master/topics/shell),
+  que es un programita que interpreta líneas de comando (command-line
+  interpreter) así como tener [git](https://github.com/Laboratoria/bootcamp/tree/master/topics/scm/01-git)
+  instalado. Si usas un sistema operativo "UNIX-like", como GNU/Linux o MacOS,
+  ya tienes una _shell_ (terminal) instalada por defecto (y probablemente `git`
+  también). Si usas Windows puedes usar [Git bash](https://git-scm.com/download/win),
```

Comment:
Como referencia, a día de hoy en las NodeSchool estamos recomendando:

* Windows 10: [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
* Windows anteriores a 10: la versión completa de [cmder](https://cmder.net/), que incluye Git Bash